pkg/ebpf/c/tracee.bpf.c

```diff
@@ -7594,6 +7626,14 @@ CGROUP_SKB_HANDLE_FUNCTION(proto)
             return CGROUP_SKB_HANDLE(proto_icmpv6);
     }
 
+    // TODO: If cmdline is tracing net_packet_ipv6 only, then the the ipv4
```

Comment:
I also quoted this in the git log, and will adjust this once we implement capturing different artifacts per given scope (and when I start to run filtering in the network events pipeline, from its perfbuffer).